lib/services/trustedcluster.go

```diff
@@ -170,16 +170,21 @@ func UnmarshalTrustedCluster(bytes []byte, opts ...MarshalOption) (types.Trusted
 
 // MarshalTrustedCluster marshals the TrustedCluster resource to JSON.
 func MarshalTrustedCluster(trustedCluster types.TrustedCluster, opts ...MarshalOption) ([]byte, error) {
+	// DELETE IN(7.0)
```

Comment:
Should this be `DELETE IN(8.0)`? I mean, this is *for* 7.0, yeah?

Reply:
Looks like a merge conflict so it should be deleted, thanks for catching this!